KomaMRIFiles/Project.toml

```diff
@@ -7,21 +7,21 @@ version = "0.7.7"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+KomaMRIBase = "d0bc0b20-b151-4d03-b2a4-6ca51751cb9c"
 MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
 MRIFiles = "5a6f062f-bf45-497d-b654-ad17aae2a530"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
-KomaMRICore = "4baa4f4d-2ae9-40db-8331-a7d1080e3f4e"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Scanf = "6ef1bc8b-493b-44e1-8d40-549aa65c4b41"
 
 [compat]
 FileIO = "1"
 HDF5 = "0.16, 0.17"
 JLD2 = "0.4"
```

Comment:
Is this being used?

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/c621cfe8372eb1437a1ebec60f55e001c88d0460